app/main/forms/work_order_forms.py

```diff
@@ -16,3 +17,19 @@ def __init__(self, brief_id, data_api_client, *args, **kwargs):
 
         responses = data_api_client.find_brief_responses(brief_id)
         self.seller.choices = [(br['supplierCode'], br['supplierName']) for br in responses['briefResponses']]
+
+
+class WorkOrderQuestionForm(DmForm):
```

Comment:
I've been taking a closer look at the WTForms docs, and I think this approach is okay.
http://wtforms.simplecodes.com/docs/0.6.1/fields.html

WTForms gives us most of what we need out of the box, and we can extend it.  For example, if we want a price field, we can make one: http://wtforms.simplecodes.com/docs/0.6.1/fields.html#custom-fields

We can also compose forms:
http://wtforms.simplecodes.com/docs/0.6.1/fields.html#field-enclosures
(Useful if we need to edit multiple fields on one page, for example -- like the "multi-questions" in the content loader.)

One caveat is that `DmForm` adds the `csrf_token`.  If you want to compose forms, you'll want to refactor things so that you can get fields without the token (otherwise you'd have to add several `csrf_token`s to a form).